### PR TITLE
Removes singularity, tesla, TEG and Mrs. Pacman. Changes supermatter overcharge delam.

### PR DESCRIPTION
**Pull request by CRITAWAKETS**
## About The Pull Request

This PR removes the tesla and Mrs. Pacman from the game. The title is actually misleading, the TEG and singularity are still in the game but inaccessible and the singularity no longer has its generator.

Field generators and tesla coils have been kept because shocking your fellow man is the greatest sensation of power in the world.

The overcharging delamination for the supermatter has been replaced by the release of 2500 mols of tritium at 500,000 kelvin. An engineer who's really hateful of the station can use this for fusion, if they desire. Otherwise, its a tritium fire in engineering alongside the explosion. This will probably be changed in the future to be more potent since the explosion will pretty much just space all the tritium away.

Singularities no longer produce radiation. Now, keeping one is less about making power and more about making a statement, especially since you need to delaminate an SM shard for it.

## Why It's Good For The Game

Having power optimized for one generator (SM+Turbine) will allow balancing to be feasible, also oranges wanted this. The other generators may later return but with tweaked values and different roles.

## Changelog
:cl:
del: Nanotrasen has removed their outdated teslas, singularities, TEGs and Mrs. Pacman generators from their servers and warehouse.
/:cl:

**DO A TESTMERGE BEFORE A FULLMERGE AS THIS COULD BREAK THINGS, EVEN IF I TESTED IT A BIT.**

### CODE REVIEW

**Comment by CRITAWAKETS** [comment]
goddamit gitkraken why do you love having a wall of commits

**Comment by TiviPlus** [comment]
Because you cant rebase properly

**Comment by CRITAWAKETS** [comment]
wait dont merge this yet, breaks charliestation

**Comment by stylemistake** [comment]
TEG is unbalanced, but it's not a reason to remove it. It's fun.

Charge delamination is also fun.

Fun apparently is not allowed.

**Comment by CRITAWAKETS** [comment]
> TEG is unbalanced, but it's not a reason to remove it. It's fun. Charge delamination is also fun.
> 
> Fun apparently is not allowed.

The TEG had to go, im sorry. I will try to make it return however, as a sidegrade to turbines for the SM once it uses heat.

Also, charge delaminating the SM into producing a tesla was fun however teslas grind the server down into powdered silicon so it had to go. That's why the singularity delamination is still there.

**Comment by Time-Green** [comment]
How are people supposed to start the station when there's no power? Or when the SM is destroyed? You can't even supply a little bit of power during chaos without ordering a new SM and rebuilding all of engineering.

**Comment by CRITAWAKETS** [comment]
> How are people supposed to start the station when there's no power? Or when the SM is destroyed? You can't even supply a little bit of power during chaos without ordering a new SM and rebuilding all of engineering.

Solars and the two pacmans that are actually useful for anything other than breaking the power system are still there. And the turbine too, get learning!

**Comment by CRITAWAKETS** [comment]
Alright, you can merge now, charlie station now has an SM shard and the supplies required to use it. You will have to get creative with your design if you dont want to expand the station's area however but it is possible (smallest SM is 2x4).

**Comment by Tharcoonvagh** [comment]
Time to never set up the engine ever again

**Comment by stylemistake** [comment]
This does nothing towards implementing the power design doc, only removes fun content and decreases replayability. Implement the design doc, then we'll talk.

**Comment by CRITAWAKETS** [comment]
> This does nothing towards implementing the power design doc, only removes fun content and decreases replayability. Implement the design doc, then we'll talk.

This is a pre-requisite to actually doing anything relating to the design doc besides the SM changes. If you added literally any change to power before this change, a simple tesla or TEG could just break the entire balance you set up.

**Comment by FloranOtten** [comment]
How about removing them in THAT PR then? 

Why the fuck are you removing the tesla delam? No fun allowed, it seems. Yeah, it grinds the server down but it's a round ending event regardless. You're only stuck with it for at most 10 minutes. Usually less, given that the shuttle will be called beforehand. 

Why are you removing the singularity? It's the easiest of them all to balance! It uses the same radiation output of the SME. Just change the level 3 or 4 to output the same radiation as a default n2 supermatter. You have now fixed the power problem WITHOUT removing a chunk of content.

**Comment by EgorDinamit** [comment]
Hm yes, based removal PR.

If there was an intent to replace singularity, tesla and TEG, then do it, not just remove it and say "Will fix this stuff in the next year".

**Comment by Kaffe-work** [comment]
Ah yes, the removal of fun things to do as a variation, instead of improving systems.

I will make power interesting in "the future"

**Comment by CRITAWAKETS** [comment]
If i put in the power-consuming stuff in right now it would be a bit too much in one PR.

**Comment by EgorDinamit** [comment]
It's never too much for one PR as long as it is in the same category.

**Comment by Tlaltecuhtli** [comment]
bait pr, sage in all fields

**Comment by CRITAWAKETS** [comment]
"So crit, why did you not optimize the tesla?"
Ah, i'll answer mystery commenter which is actually someone.

Its simple really. The tesla lags because of 4 things. The zaps (These can be optimized but these arent the main source of lag), the explosions (these were recently optimized however you still get noticeable stutter), the atmos changes (ah good let me just recode the entire atmos system, it will actually be worse with monstermos because you constantly have to recalculate big changes) and the gigantic amount of items from the explosions breaking things (ah yes let me just recode byond for you no pressure). So yeah, to optimize the tesla to not make the server slow down to a crawl, the game engine has to be replaced.

Also for the second part, im a trash coder. I can barely code anything and its a wonder i managed to do this without accidentally opening rifts in reality or breaking everything.

**Comment by Kaffe-work** [comment]
One of the worst crimes here is the lighting system as well - as said before, having the tesla cause a global blackout would improve this.

**Comment by GuillaumePrata** [comment]
Guess we could make Tesla not explode things enough to make hull breaches, just damage machinery and zap carbons to death.
I understand the reason and hate Tesla lag, but I like the Tesla delam soo... ye...

**Comment by nianjiilical** [comment]
what does this actually add to the game in exchange for removing sm delam as a method for traitors to end the round

**Comment by brotherbeyondd** [comment]
Its meant to be the start of something great, but it's far easier to remove things than to add new things so it'll be like the RD console being moved (?) and nothing will come of it for ages.

**Comment by TheMidnightRose** [comment]
As someone who does play engineering. I am fine with Singularities creating reduced radiation, Nerfing the TEG output is actually a good idea as well, since TEGs have always been horribly broken. 

Removing Tesla because lag? Really? If we removed everything that caused some lag we wouldnt have a game anymore to play. Optimize it more then. We got the Arcs and Zap bugs fixed so they dont continue infinitely anymore. We have optimized the SM code enough that it isnt held together with hopes and dreams.
Why is it that optimizing the Tesla so hard that you decide to remove its existence?

**Comment by optimumtact** [comment]
`Removing Tesla because lag? Really? If we removed everything that caused some lag we wouldnt have a game anymore to play. Optimize it more then.`

Nobody will

**Comment by Winterous** [comment]
I don't understand why the Mrs Pacman is thrown in here.

**Comment by AutisticFroggy** [comment]
> Alright, you can merge now, charlie station now has an SM shard and the supplies required to use it. You will have to get creative with your design if you dont want to expand the station's area however but it is possible (smallest SM is 2x4).

Smallest SM is 2x3 and is pretty stable so it should be doable.

> Removing Tesla because lag? Really? If we removed everything that caused some lag we wouldnt have a game anymore to play. Optimize it more then.

> Nobody will, it's been known to lag for about a year + and still nobody has ever touched it.

A halfway solution that has been implemented on a downstream for quite some time and works pretty well has been to cap the number of miniballs to 10. Actually optimizing the explosions and special sprite drawing procs to make it not crash client and server just takes too much effort so this solution was implemented in the interim.

File: code/modules/power/supermatter/supermatter.dm [hunk 1]
```diff
@@ -440,10 +440,9 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			S.consume(src)
 			return //No boom for me sir
 	else if(power > POWER_PENALTY_THRESHOLD)
-		investigate_log("has spawned additional energy balls.", INVESTIGATE_SUPERMATTER)
+		investigate_log("has released tritium from an overcharge.", INVESTIGATE_SUPERMATTER)
 		if(T)
-			var/obj/singularity/energy_ball/E = new(T)
-			E.energy = power
+			atmos_spawn_air("tritium=2500;TEMP=500000") //extremely fucking hot and could do fusion if you tried hard enough //GEORGE SEARS DID 9/11
```
Comment (LemonInTheDark) [inline code/modules/power/supermatter/supermatter.dm:445]: Have you tested this? The engine will just blow up and space all this gas immediately

**Comment by AtlasRules** [comment]
and this is why nobody likes /tg/ aside from stupid people

**Comment by LemonInTheDark** [comment]
> A halfway solution that has been implemented on a downstream for quite some time and works pretty well has been to cap the number of miniballs to 10. Actually optimizing the explosions and special sprite drawing procs to make it not crash client and server just takes too much effort so this solution was implemented in the interim.

The main issue here is a lack of data of what causes the lag. I can tell you it's not primarily explosions since we got goofbooms

**Comment by LemonInTheDark** [comment]
> 
> 
> `Removing Tesla because lag? Really? If we removed everything that caused some lag we wouldnt have a game anymore to play. Optimize it more then.`
> 
> Nobody will, it's been known to lag for about a year + and still nobody has ever touched it.
> 
> if someone wants to fix it, they can do it in a re adding PR where they prove they've actually cleaned it up and it doesn't lag and it's well put together.

Aight, I'll get some profiles of it and do what I can

**Comment by edgeotron** [comment]
why is oranges wanting anything even concievable as a plus

**Comment by san7890** [comment]
> The overcharging delamination for the supermatter has been replaced by the release of 2500 mols of tritium at 500,000 kelvin.

Toxins Powercreep.

**Comment by necromanceranne** [comment]
Were tesla coils kept mostly because of SM zaps? Because there is a lot of really fucking silly stuff you can do right now with tesla coils and things that produce a zap (themselves with a signaller, tesla guns, etc.) that produce quite a lot of power. Mostly bringing this up simply because your aim is to keep under control sources of power.

**Comment by LemonInTheDark** [comment]
Tesla guns might need to get yeeted, I'm working on fixing the coil issues myself, expect a pr soon

**Comment by Winterous** [comment]
Tesla guns have always seemed really gimmicky and just plain bad, to me.

File: code/modules/power/supermatter/supermatter.dm (hunk 1 above)
Comment (CRITAWAKETS) [reply code/modules/power/supermatter/supermatter.dm:445]: Yeah, i've been thinking on how to make it better.

**Comment by PKPenguin321** [comment]
PACMANs are good for an entirely different purpose than powering the entire station (such as temporarily powering a small space base), would be nice to see a suitable replacement

**Comment by PKPenguin321** [comment]
Other thing I'm not convinced on is the replacement delam over tesla, still seems pretty weak, wont the gas just get vented to space through the hole the SM just blew

**Comment by CRITAWAKETS** [comment]
> PACMANs are good for an entirely different purpose than powering the entire station (such as temporarily powering a small space base), would be nice to see a suitable replacement

PACMANs are still in. Just not the Mrs Pacman since it just serves to break the power system by burning diamonds (barely any aswell) without any effort whatsoever.

**Comment by CRITAWAKETS** [comment]
Also quick side note, the Super PACMAN may eventually go in a later PR of mine, however it's planned to be replaced with something which is more engaging to use and has more uses in some fashion. It may or may not also be part 1/2 of a planned engineering expansion, whichever part comes first.

File: code/modules/power/supermatter/supermatter.dm (hunk 1 above)
Comment (Fikou) [reply code/modules/power/supermatter/supermatter.dm:445]: if its an energy based delamination, why not do something like breaking all the lights, causing ion storm and disabling apc's along with the explosion

Comment (LemonInTheDark) [reply code/modules/power/supermatter/supermatter.dm:445]: at that point just keep the tesla man, it'll be cheaper.

**Comment by Kholdgrave** [comment]
So you're just removing things that generally add to rounds and have absolutely nothing to replace them with? Ripping a sink out of a bathroom isn't the same as fixing it, if you're going to remove things that are favorite events of players then at least have something substantial to soothe the immense disappointment that another classic /tg/ feature has been removed again.

**Comment by Cyprex** [comment]
>Singularities no longer produce radiation
That's unnecessary, a future PR that switches the SM over to turbines would presumably disable radiation collectors as well.

**Comment by edgeotron** [comment]
i am honestly fucking begging you to look at the ratio of likes to dislikes on this and realize every part of what you are doing will make the game worse, im so tired of tg removing fun and complex stuff man please just dont do this

**Comment by ArcaneDefence** [comment]
> Smallest SM is 2x3 and is pretty stable so it should be doable.

image
This design requires no input and will happily chug along. 

Ancient stationers that already know engineering will have no problem with space because of using an sm instead of a scrung.

File: code/modules/power/supermatter/supermatter.dm (hunk 1 above)
Comment (wesoda25) [reply code/modules/power/supermatter/supermatter.dm:445]: massive emp blast would be cool

**Comment by FloranOtten** [comment]
The more I think this over, the more I realize how stupid this entire PR is. For example, the singularity. There's two easy ways to fix  the singulo producing power; Make the thing unable/bad at producing power (in the form of radiation), or make it unable to be summoned for power production. Why both? Is there something you dlslike about the singularity? Nobody is going to risk getting banned for singuloing the station because the crystal blew up their field generators. 

I don't see why the tesla needs to go entirely, but fine. Whatever. But at least replace the delamination with something. Anything. 2500 mols of tritium? That just gets spaced instantly? Are you having a laugh? Even if it didn't explode it's worthless. The only thing you can use it for is 1. Maxcaps (or just use the atmos incinerator like a normal person), or 2. Open air fusion. This is not only a bullshit way to die for those actually in the range (oh boy time to just drop dead), it's also stupidly easy to stop. Just add some space tiles around the place. Not to mention the amount of setup required for open-air fusion, all of which needs to be done after you've been setting off stationwide alarms for 10 minutes and spent a while extinguishing the tritium merrily burning away.

What is your plan for Charlie station? Their project has always been making a singularity, to power themselves up. Are you adding more solars crates?

I just hope whatever maintainer is considering that merge button will think to wait until things like the charge delamination being pure dogshit and just hold off until it's fixed.

**Comment by optimumtact** [comment]
>What is your plan for Charlie station?
Who cares

>Muh tesla
Make a new pr adding it back and prove it doesn't lag, I'm not waiting

>singularity
It's not being removed, you just cant generate it

>Muh second meltdown
Make a pr adding a better one if you think you can

>I hope you're not considering merging
I am

**Comment by edgeotron** [comment]
when you remove legitimate complexity to the game just to get rid of some lag
image


honestly though like less than a week ago on fulp 20 tesla balls went loose and it barely lagged

**Comment by optimumtact** [comment]
ah yes, the legendary complexity of the tesla

*checks notes* an object that sits in side a laser fence and zaps tesla coils

**Comment by edgeotron** [comment]
by complexity i mean gameplay variety, important variety at that

**Comment by optimumtact** [comment]
bahahaha, get outta here

**Comment by edgeotron** [comment]
id tell you the same

**Comment by TheMidnightRose** [comment]
I actually saw a Tesla escape yesterday night with about 10-15 mini balls. I think lighting took a bit of a hit, but beyond that the game moved along perfectly fine. Time Dilation didnt go beyond 50% at its highest

**Comment by Jared-Fogle** [comment]
I don't like the Tesla being removed as much as anyone else, but come on now. Trying to deny that it lags the server is laughable.

**Comment by LemonInTheDark** [comment]
Frankly I don't think teslas that are restricted by the sm are a major concern, from my experience the major issues have been when someone spawns like 10 of them. It's still a danger tho, and could be improved by reducing the explosions and active turfs. Singulos have somewhat of the same problem, but they're more rare so it's seen less

**Comment by griffithlol** [comment]
i dont want to leave a massive dump here, but turbines need to be reworked as well. The code is spaghitti from like ten years ago, and lacks any basic understanding of how a turbine works or should look. They fucking put the compressor after the combustion chamber for fucks sake

**Comment by LemonInTheDark** [comment]
please don't start talking about pumps you'll make me cry

**Comment by griffithlol** [comment]
please look at my thermo diagrams they are very nice

**Comment by optimumtact** [comment]
where are they?

**Comment by CRITAWAKETS** [comment]
alright, fixed the merge conflict and figured out how gitkraken works a bit more

**Comment by AlucardHellsingChapel** [comment]
ah yes removing shit to  add replace in one  year

**Comment by trerri** [comment]
If you really need to do this then at least comment out the tesla instead of deleting it outright, and perhaps add a helpful little comment asking to unlag it so it doesn't stay forgotten. The charge inertia chain reaction could spawn a bunch of demon portals with pissed off ghost controllable ethereals (simplemob AI by default like blob zombies). I think that if oranges thinks it's feasible to rework all power costs to make machinery more expensive than lighting I don't think why the TEG would need to go as well. The turbine is less interesting, more exploitable, and makes no sense physics-wise.

**Comment by stylemistake** [comment]
Restating my opinion here:

- TEG should remain because it's fun to build and optimize when playing long rounds, and it can serve really well in a thermal SM setup. All it needs is reduced efficiency. Wouldn't mind even 5-10x nerfs.
- Tesla should remain as delam effect, because it's rare and I have never seen such a delam load the server for more than 50%.
- Rest of removals - don't care.

**Comment by PapaSejd** [comment]
Tg at it again, removing features and  "re-added" a year later.

I have more hope that star citizen comes out before tg re adds removed features

**Comment by Fabian272** [comment]
What a good code maintainer
"'>Whats your plan for charlie?'
Who cares"
Who cares about tg? Not you thats for sure. Hey why not remove the spawner, ez fix
And regarding comment above
Yandere sim will be out before tg re-adds removed stuff

**Comment by Spepbuthewranglesgithub** [comment]
and so the cycle of removing, promising to fix/replace, and never actually fixing/replacing it afterwards strikes again.

is it really that hard to just replace everything at once like the lavaland PR, we still _could_ have a reworked tesla that actually takes more brainpower to get running than a saltpetre factory. _even though the boring crystal is about as easy._

_albeit the idea of ""balancing"" the SM terrifies me to no end with SM delams already meaning an automatic shuttle call, nevermind making it easier to delam if you manage to start it up at all, not a good implication_

plus, why remove the Mrs. Pacman. that just seems unnecessary.

**Comment by WarlockD** [comment]
Well I am happy for the shard in charlie.  It was a pain in the ass to ever set up that singularity.   While I understand removing the tesla I am sure the TEG will come back if people are worried.  It's just a matter of making it dangerous.  I mean most of the arguments here about keeping it is its  a "safe" way to have unlimited power.

This is not the SS13 way.  Things need to blow up, especially something that can produce millions of watt with no apparent downside.

**Comment by CRITAWAKETS** [comment]
The TEG is intentionally rendered unobtainable but not removed because it may actually come back in some fashion. Im thinking as some sort of alternative to turbines for the SM when it does use turbines.

**Comment by AlucardHellsingChapel** [comment]
> Well I am happy for the shard in charlie. It was a pain in the ass to ever set up that singularity. While I understand removing the tesla I am sure the TEG will come back if people are worried. It's just a matter of making it dangerous. I mean most of the arguments here about keeping it is its a "safe" way to have unlimited power.
> 
> This is not the SS13 way. Things need to blow up, especially something that can produce millions of watt with no apparent downside.

well  i set up as engie borg w ith 2 more ppl on charlie  singulo in 5 minutes

**Comment by optimumtact** [comment]
Conflicts

**Comment by JenqaDev** [comment]
If this bs gets merged I'm going to rethink why I play on /tg/.
> Fun apparently is not allowed.
This sentence basically explains the tg codebase
die orangeman

**Comment by JenqaDev** [comment]
someone take this PR down and the clown get some fresh air for a while

**Comment by FloranOtten** [comment]
> It's not being removed, you just cant generate it outside of overloading the SM

That's the fucking problem, though. Why? What does it change for the better? What do we lose by keeping it in the game, unable to produce radiation? Why would ANYONE setup such a large setup, for what? A level 1 singularity? You can't even grow it, because you've gone and removed the damn particle accelerator. 

> Make a pr adding a better one if you think you can
"Hey, please don't replace your back garden with an open septic system"
"Fucking go hire a landscaping firm if you want something better, then"

> Who cares
"Fuck content."

**Comment by JenqaDev** [comment]
careful floran, you might also get marked as disruptive

**Comment by LittleNyanCat** [comment]
It has been fun seeing /tg/ going downhill for the last year, thank god bee exists

**Comment by 1glitchycent** [comment]
local """"good"""" engineer main just fucking the majority of power generation for no reason besides /tg/ coderbus not wanting to actually code improvements lol

**Comment by CRITAWAKETS** [comment]
> local """"good"""" engineer main just fucking the majority of power generation for no reason besides /tg/ coderbus not wanting to actually code improvements lol

hey oranges found a boomer

**Comment by PapaSejd** [comment]
I think your almost 1:100 ratio in likes to dislikes is telling about the state of your PR.

**Comment by uBadush** [comment]
yeahi I'am happy beestation exist, removal culture is against everything space station 13 is about, and tg is following it, orange man bad unironicaly

**Comment by Fabian272** [comment]
Based

**Comment by MACIEKBAKI** [comment]
There is one thing every /tg/ coder loves; removals.

**Comment by AffectedArc07** [comment]
Put me in the screenshot

**Comment by Iamgoofball** [comment]
The tesla has been fixed in https://github.com/tgstation/tgstation/pull/52930. Remove the tesla removal from this PR.

**Review by stylemistake — CHANGES_REQUESTED** [review]
As goof said, remove the tesla removal from this PR, and I also strongly suggest unremoving TEG and nerfing it instead.

**Comment by PapaSejd** [comment]
> The tesla has been fixed in https://github.com/tgstation/tgstation/pull/52930. Remove the tesla removal from this PR.

Lol, do you honestly think OP is removing teslas and shit cause of lag or some reason like that? This is just another """coder""" whos never played the game and has only tha ability to remove and not improve. Tg will remove until theres just a lobby screen.

**Comment by Iamgoofball** [comment]
@PapaSejd hey buddy I think you got the wrong github, hippie station is two repos down

**Comment by PapaSejd** [comment]
@Iamgoofball No im quite sure, I believe its been TGstation that has been removing large amounts of features recently. Good thing I dont play on a tg downstream of any sort

**Comment by Muncher2112** [comment]
@PapaSejd Then why are you here

**Comment by PapaSejd** [comment]
@Muncher2112 came to see how downhill tg has gotten since I stopped developing for its downstreams a bit ago. Truly is entertaining

**Comment by Muncher2112** [comment]
@PapaSejd That's kind of sad to be honest.  Do you at least charge rent for the space in your head?

**Comment by JenqaDev** [comment]
@PapaSejd where do you play? maybe I'm considering going there if it's not skyrat

**Comment by LittleNyanCat** [comment]
Boys, we officially have reached a 1:100 like-to-dislike ratio. yikes

**Comment by PapaSejd** [comment]
@JenqaDev I have a server I work for, but other than that I play on beestation downstreams like AU. Im not an australian mind you, they just have a superb codebase.

**Comment by Iamgoofball** [comment]
chrome_coyUL56QHG
chrome_VNCGe2ioCf
180c0f3e052e4832-thinking-emoji-gif-gif-images-download

**Comment by PapaSejd** [comment]
@Iamgoofball yea yea, beestation is a tg downstream I know, but you know what I mean by "tg-downstreams". Stuff like fulp, yog, etc.

**Comment by JenqaDev** [comment]
beestation downstreams got me there, respect went from 100 to 0 rather fast. 
just say degenerated tg

**Comment by JenqaDev** [comment]
god bless, you're clear. respect back to 100

**Comment by genessee596** [comment]
Why, though?

Why does this PR need to exist? I see no issue with functional singulos' and teslas. Hell, at this point, they're part of /tg/ culture. Why remove them? 

Sure, the TEG exists - I have almost never seen that used. And, again - the singulo and tesla have no reason to be removed. Forcing the playerbase to exactly two different forms of efficient engines instead of four is just unnecessary.

The Mrs. Pacman doesn't need to exist, though, and I support the removal of that. Have never seen that generator used, and I doubt it ever will be used.

**Comment by optimumtact** [comment]
goof can readd the tesla in a whole PR

**Comment by TreyWJ** [comment]
"Having power optimized for one generator (SM+Turbine) will allow balancing to be feasible"

I don't understand. Why remove fun singulo/tesloose shenanigans from cargo? What's that have to do with power optimization and balancing?

**Comment by Featherfield** [comment]
Based retards

**Comment by LittleNyanCat** [comment]
this is so sad. alexa, play despacito

**Comment by wesoda25** [comment]
lmao they actually merged it

**Comment by d3athrow** [comment]
lol

**Comment by PrimeDSS13** [comment]
dear lord

**Comment by AlucardHellsingChapel** [comment]
#Annoy Oranges

**Comment by ancientpower** [comment]
based

**Comment by LoserWasTaken** [comment]
i love tg

**Comment by AlucardHellsingChapel** [comment]
Uncool

**Comment by boy2mantwicethefam** [comment]
Joke server

**Comment by TobyThorne** [comment]
lmao wtf
Screw teslas and TEGs, but removing singulo is the same as cutting clowns and toolboxes. Is there any real reason to do so?

**Comment by MajorMeltdown** [comment]
soon c*ders will remove everything fun and we will turn into a lifeweb clone of sorts

**Comment by LittleNyanCat** [comment]
just wait till ss13 turns into a shitter, 2d version of second life

**Comment by PavelCIA** [comment]
my wifes son approves of this PR

**Comment by gurfan** [comment]
based

**Comment by Archanial** [comment]
This is the most unwanted pr of 2020 it was obvious oranges would merge it.

**Comment by msgerbs** [comment]
So, literally nobody but the person who submitted this gave any support to the idea. And yet the maintainers just wordlessly merged it. The only justification was "lol it makes balance easier", but there's no actual balancing done here. Why not make a PR, ya know, balancing the other power methods? Or at least actually do the balancing you talk about, because frankly I can't imagine any sort of "balance" to be had here that isn't just "do the meta improvements to power or the station runs out of power every shift". Removing something with the vague promise of future balance, and "because a maintainer known for doing things nobody wants wants it" seems like extremely poor reasoning.

**Comment by DamianX** [comment]
based

**Comment by TreyWJ** [comment]
Yeah I'm kinda surprised these go through when the entire userbase doesn't want it.

**Comment by qwertyquerty** [comment]
😳

**Comment by msgerbs** [comment]
> 
> 
> Yeah I'm kinda surprised these go through when the entire userbase doesn't want it.

I wish I could say the same.

**Comment by SweptWasTaken** [comment]
It's been posted to Reddit i'd lock it

**Comment by optimumtact** [comment]
what's Reddit?

**Comment by uBadush** [comment]
The place you belong, come home orange man

**Comment by Tharcoonvagh** [comment]
Screenshot_20200815_073430
Gotta love it!

**Comment by Tharcoonvagh** [comment]
Wait whats the point of the susp beacon if theres nothing to attract

**Comment by Tharcoonvagh** [comment]
Can you not merge terrible PRs like this? With the cloning one I agreed since it was cancer but this removes a major part of the game in favour of better framerate

**Comment by Tharcoonvagh** [comment]
Was it worth it tho?

**Comment by actioninja** [comment]
I'm not sure about this PR itself but github isn't a good place for further discussion. Make a forum thread.

**Pull request merged**